sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,428 +1,428 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.5.0 (2022-03-31)
 
 ### Features Added
 
-- When a `RequestFailedException` exception is thrown, the `what()` method now includes information about the HTTP request which failed.
```

Comment:
Why the change from '-' to '*'

Reply:
:(. Should've came from some format extension from VSCode..  I need to revert it